src/web/arr/trove/chart.arr

```diff
@@ -1046,6 +1046,8 @@ type IntervalChartSeries = {
   colors :: Option<RawArray<I.Color>>,
   pointers :: Option<RawArray<Pointer>>,
   pointer-color :: Option<I.Color>,
+  point-size :: Number,
+  lineWidth :: Number,
```

Comment:
For consistency's sake, shouldn't this be `line-width`?

Reply:
Unfortunately, the existing series use camelCase and kebab-case for different fields, and I was trying to be consistent with them.

Reply:
makes sense. @jpolitz any interest in Dorai opening another PR in which these get normalized across the charting library?

Reply:
No issue with it, though having it separate is preferred.

Reply:
(Separate PR I mean, not separate naming conventions :-))